### DISCUSSION TITLE
How do the modules gruntwork is offering differ from the modules already available via the terraform community?

### DISCUSSION THREAD

**Discussion by marijakstrazdas** [gruntwork-io/knowledge-base#117] [Help] [answered]
How do the modules gruntwork is offering differ from the modules already available via the terraform community, from a technical perspective?

**Comment by brikis98** [comment, marked as answer]
This is something we cover in the [Gruntwork FAQ](https://gruntwork.io/faq#gruntwork-vs-the-competition-at-a-glance)!
